pkg/api/handlers/toolreferences.go

```diff
@@ -166,6 +170,10 @@ func (a *ToolReferenceHandler) Delete(req api.Context) error {
 		return types.NewErrBadRequest("cannot delete builtin tool reference %s", id)
 	}
 
+	if !toolRef.Spec.Bundle {
```

Comment:
I'm not seeing anywhere where this is getting set to `true`.

Reply:
missing some files when pushing... it is set here https://github.com/obot-platform/obot/pull/1633/files#diff-fc4ca6da48915751ba6a956a3d3912415270a3c9374355c8ea3d6f788b6e222dR47